recipes/r-qs2/meta.yaml

```
@@ -0,0 +1,108 @@
+{% set version = '0.1.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-qs2
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/qs2_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/qs2/qs2_{{ version }}.tar.gz
+  sha256: c2dec8a8643db08ae6e496fd775fe723aba86742d9fb3fc93b65aa640e162f04
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
```

I believe you can whitelist libtbb because it gets picked up via a non-standard path (i.e., vendored by r-rcppparallel):

```suggestion
    - '*/Rlapack.dll'  # [win]
    - '*/libtbb.*'
```

---

Also, I'm not sure why the Linux version doesn't shows that it detects TBB, but then doesn't include a `-ltbb` flag when linking. Maybe that's an issue with upstream?